src/smc-util-node/test/misc_node-test.coffee

```diff
@@ -48,4 +48,25 @@ describe "sanitizing HTML", ->
+
+describe 'check sales tax work', ->
+    sales_tax = misc_node.sales_tax
+
+    it 'knows about 98122', ->
+        expect(sales_tax('98122')).toBe 0.10100
+    it 'knows all expected zip codes', ->
+        for zip in WA_zips
+            v = sales_tax(zip)
+            expect(v).toBeGreaterThan(0)
```

Comment:
sales tax is the sum of the state rate and a local rate.  The WA state rate is 6.5%.
See http://www.taxrates.com/state-rates/washington/?referrer=https://www.google.com/&lastReferrer=www.taxrates.com&sessionId=1492826583854
So, change this > 0 to >= 6.5%.